apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,182 @@
+description: 'In this blog, we will go through the steps to complete a sprint zero of setting up XM Cloud.  In addition to learning the basics of XM Cloud, you will become familiar with setting up a site, setting up the dev environment to support building that site, and how to connect your codebase that runs on your local rendering host to the preview endpoint of XM Cloud.'
+promoAfter: ['learning-essentials']
+openGraphImage: 'https://mss-p-006-delivery.stylelabs.cloud/api/public/content/1915549492604a64864578fe51d2a597?v=244f9e64'
+---
+
+## Introduction - What You are Going to Learn 
+
+In this blog, we will go through the steps to complete a sprint zero of setting up XM Cloud.  In addition to learning the basics of XM Cloud, you will become familiar with setting up a site, setting up the dev environment to support building that site, and how to connect your codebase that runs on your local rendering host to the preview endpoint of XM Cloud.  
+
+## Overview  
+
+XM Cloud focuses on being a headless, enterprise content and website management system.  You can manage XM Cloud via several UIs, and also through APIs and the CLI. For this blog, we will focus on the XM Cloud deploy UI for building and launching a site with XM Cloud. ( For more information on APIs with XM Cloud, navigate [here](https://doc.sitecore.com/xmc/en/developers/xm-cloud/xm-cloud-deploy-api.html).) 
+ 
+## Setup the project and environment  
+
+Now, we will setup the developer project and environment that we will create therein. Let’s get started! In the first stage, we will access the cloud portal and create a new *project*.  
+
+### Log into Sitecore Cloud Portal 
+
+1. To begin, open the Sitecore Cloud Portal (https://portal.sitecorecloud.io) and log in.   
+2. Access the XM Cloud project and environment management interface by clicking `XM Cloud Deploy`.  
+![Sitecore Portal - Open XM Cloud Deploy](https://sitecorecontenthub.stylelabs.cloud/api/public/content/42edc85bc7bd41e2914f29b7b260496b?v=94261954)
+
+### A Quick Dive into Projects and Environments  
+
+Let’s take a brief pause and discuss how projects and environments work together in XM Cloud. A project is connected to a source code repository, for example, GitHub.  
+
+![A GitHub repository is mapped to a project in XM Cloud](https://sitecorecontenthub.stylelabs.cloud/api/public/content/142a8648763a42eda1a210b1488cfda5?v=859d5477) 
+
+Each project can have multiple environments. A typical setup is to have one source code repository for one Brand or legal entity, and then have a DEV environment, QA, Staging or Pre Prod, and Production. 
+
+![XM Cloud Projects and Environments](https://sitecorecontenthub.stylelabs.cloud/api/public/content/9deae49bb7f5428285baf2ea374fe1ec?v=40ffa388)
+ 
+We will start with a Dev Environment for now, and create the other environments later. Let’s get back to project creation! 
+
+### Create a new Project and Authoring Environment 
```

Comment:
I'd put the first step, 'Login into Sitecore Cloud Portal' (line 20) here, after the 'A Quick Dive...' (lines 26-37)

Reply:
This seems to be resolved